src/app/views/sap/register.php

```diff
@@ -63,13 +63,13 @@
             <?php
             if (isset($errors) && count($errors) !== 0) {
                 foreach ($errors as $error) {
-                    echo "<p class=\"error\">$error</p>";
+                    echo "<p class=\"text-error\">$error</p>";
                 }
             } else if (isset($success)) {
                 if ($success) {
-                    echo "<p class=\"success\">You have applied successfully. We shall review your application and get back to you.</p>";
+                    echo "<p class=\"text-success\">You have applied successfully. We shall review your application and get back to you.</p>";
```

Comment:
Nice observation. :D Impressed by your reviewing. :P But yeah, I'm aware of this, and yeah, sure about this. :) They're not different enough to cause any issues in the application.